workflow/executor/resource.go

```diff
@@ -97,7 +97,8 @@ func (we *WorkflowExecutor) getKubectlArguments(action string, manifestPath stri
 	if len(buff) != 0 && action != "patch" {
 		args = append(args, "-f")
 		args = append(args, manifestPath)
-	} else if len(flags) <= 0 {
+	}
+	if len(flags) == 0 && len(buff) == 0 {
```

Comment:
Minor: should this block be placed at the top of this func before collecting args, as another validation of user inputs?